### PR TITLE
docs(angular): add angular framework integration document

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ part of elastic/apm-agent-rum-js#292

### CODE REVIEW

**Comment by vigneshshanmugam** [comment]
Thanks @bmorelli25

**Comment by vigneshshanmugam** [comment]
will merge after #384.

File: docs/angular-integration.asciidoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+@NgModule({
+  imports: [BrowserModule, RouterModule.forRoot(routes)],
+  declarations: [AppComponent, ContactListComponent, ContactDetailComponent],
+  providers: [ApmService],
+  bootstrap: [AppComponent]
+})
+export class AppModule {
+  constructor(service: ApmService) {
+    service.init({
```
Comment (jahtalab) [inline docs/angular-integration.asciidoc]: Maybe we should document the apm instance here!

```
var apm = service.init()
apm.setUserContenxt()
```

File: docs/angular-integration.asciidoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+[source,bash]
+----
+npm install @elastic/apm-rum-angular --save
+----
+
+[float]
+==== Instrumenting your Angular application
+
+The Angular integration package comes with a `ApmService` that will start subscribing to 
+https://angular.io/api/router/Event[Angular Router Events] once the service is initialised.
```
Comment (axw) [inline docs/angular-integration.asciidoc]: s/initialised/initialized/ ?
(we should be consistent one way or the other; using z below)

File: docs/angular-integration.asciidoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+})
+export class AppModule {
+  constructor(service: ApmService) {
+    // API is exposed through this apm instance
+    const apm = service.init({
+      serviceName: 'angular-app',
+      serverUrl: 'http://localhost:8200'
+    })
+
+    apm.setUserContenxt({
```
Comment (axw) [inline docs/angular-integration.asciidoc]:
```suggestion
    apm.setUserContext({
```

File: docs/angular-integration.asciidoc (hunk 2 above)
Comment (bmorelli25) [reply docs/angular-integration.asciidoc]: Good catch. "Initialized" is Standard American English (what we use in the docs) so this should be changed.

Comment (vigneshshanmugam) [inline docs/angular-integration.asciidoc]:
```suggestion
https://angular.io/api/router/Event[Angular Router Events] once the service is initialized. 
```

Comment (vigneshshanmugam) [reply docs/angular-integration.asciidoc]: Thanks @axw and @bmorelli25

**Pull request merged**